app/controllers/concerns/batch_processable.rb

```diff
@@ -1,14 +1,8 @@
 module BatchProcessable
   extend ActiveSupport::Concern
 
-  def load_selected_objects(klass)
-    klass = klass.accessible_by(current_ability)
-    if params[:select_all].present?
-      klass.all.to_a
-    elsif params[:selected].present?
-      params[:selected].keys.collect{ |id| klass.find_by_id(id) }.compact
-    else
-      []
+  def load_selected_objects(rel)
```

Comment:
Comment: "Restricts the given scope to the specific objects checked by the user, if any. If no boxes checked, or if the `select_all_pages` option is set, returns the scope unchanged."

Reply:
Maybe rename this method to `restrict_scope_to_selected_objects`

Reply:
done